interface/ceed-qfunction.c

```diff
@@ -681,6 +617,70 @@ int CeedQFunctionAddOutput(CeedQFunction qf, const char *field_name,
   return CEED_ERROR_SUCCESS;
 }
 
+/**
+  @brief Get the CeedQFunctionFields of a CeedQFunction
+
+  @param qf                  CeedQFunction
+  @param[out] input_fields   Variable to store input_fields
+  @param[out] output_fields  Variable to store output_fields
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref Backend
+**/
+int CeedQFunctionGetFields(CeedQFunction qf, CeedQFunctionField **input_fields,
+                           CeedQFunctionField **output_fields) {
```

Comment:
Number of fields?

Reply:
We made those two separate functions originally for... reasons? I agree that extra outputs make sense here.